### ISSUE TITLE
“Boot dev” failes to boot

### ISSUE THREAD

**Issue by stewdio** [jackrusher/geometer#3]
When running the command `boot dev` (or `boot run`, etc.) from the command line, I receive the following error:
```
java.lang.IllegalStateException: template already refers to: #'boot.core/template in namespace: adzerk.boot-reload
     clojure.lang.ExceptionInfo: template already refers to: #'boot.core/template in namespace: adzerk.boot-reload
    line: 1
```
As I’m new to this, I don’t know how to resolve it on my own 😅
But I imagine this error is due to some upstream update that’s happened since this repo was last touched?

**Comment by jackrusher** [comment]
Indeed, upstream changes in a third party build tool have broken this repo. I've got a TODO to switch to the standard Clojure build/deps system, but haven't gotten around to it yet. This ticket and interest on Twitter have led me to raise the priority on that, so I should get to it within a week or so. :)

**Comment by MrGung** [comment]
I switched to deps/bb on my fork: https://github.com/MrGung/geometer/tree/removing-boot

There is a runtime-error, though, which I couldn't get rid of...

**Comment by jackrusher** [comment]
@MrGung I've also switched to deps, but there are still problems. The version of `geom` used here does something that's illegal in later versions of the ClojureScript compiler, which are expected by Figwheel and such. Unfortunately, upgrading `geom` exposed a different breaking change in how WebGL buffers are handled. I have not yet had time to read the source to figure out what I need to change to make this code work with that version.
